taca/element/Element_Runs.py

```diff
@@ -0,0 +1,1227 @@
+                    idx1_overlapped_len = min(
+                        [
+                            demux_lens_pair[1]
+                            for demux_lens_pair in sub_demux_index_lens
+                            if demux_lens_pair[0] == sub_demux
+                        ][0][0],
+                        [
+                            demux_lens_pair[1]
+                            for demux_lens_pair in sub_demux_index_lens
+                            if demux_lens_pair[0] == sub_demux_with_max_index_lens
+                        ][0][0],
+                    )
+                    idx2_overlapped_len = min(
+                        [
+                            demux_lens_pair[1]
+                            for demux_lens_pair in sub_demux_index_lens
+                            if demux_lens_pair[0] == sub_demux
+                        ][0][1],
+                        [
+                            demux_lens_pair[1]
+                            for demux_lens_pair in sub_demux_index_lens
+                            if demux_lens_pair[0] == sub_demux_with_max_index_lens
+                        ][0][1],
+                    )
+                    for unassigned_index in unassigned_indexes:
+                        idx1_overlapped_seq = unassigned_index["I1"][
+                            :idx1_overlapped_len
+                        ]
+                        idx2_overlapped_seq = unassigned_index["I2"][
+                            :idx2_overlapped_len
+                        ]
+                        # Remove the overlapped record from the max_unassigned_indexes list
+                        max_unassigned_indexes = [
+                            max_unassigned_index
+                            for max_unassigned_index in max_unassigned_indexes
+                            if not (
+                                max_unassigned_index["I1"][:idx1_overlapped_len]
+                                == idx1_overlapped_seq
+                                and max_unassigned_index["I2"][:idx2_overlapped_len]
+                                == idx2_overlapped_seq
+                            )
+                        ]
+            # Append to the aggregated_unassigned_indexes list
+            aggregated_unassigned_indexes += max_unassigned_indexes
+        # Sort aggregated_unassigned_indexes list first by lane and then by Count in the decreasing order
+        aggregated_unassigned_indexes = sorted(
+            aggregated_unassigned_indexes, key=lambda x: (x["Lane"], -int(x["Count"]))
+        )
+        # Fetch PFCount for each lane
+        pfcount_lane = {}
+        if os.path.exists(self.run_stats_file):
+            with open(self.run_stats_file) as stats_json:
+                aviti_runstats_json = json.load(stats_json)
+            for lane_stats in aviti_runstats_json["LaneStats"]:
+                pfcount_lane[str(lane_stats["Lane"])] = float(lane_stats["PFCount"])
```

Comment:
@chuan-wang, this is also affected by the `lane_stats["Lane"] = "0"` bug  that we discussed.

Reply:
@alneberg Fixed in this PR: https://github.com/ssjunnebo/TACA/pull/18